recipe/get_libgdal_version.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from subprocess import check_output
+import json
+import os
+
+
+if __name__ == '__main__':
+    conda_args = ['conda', 'list', '-p', os.environ['PREFIX'], 'libgdal', '--json']
```

Comment:
Thanks! A long time ago I had an idea about something similar but using gdalinfo instead, never got to it though. This is OK to be merged but do you want to try gdalinfo first? The main advantage is to not rely on `conda list` and the PREFIX env var, which may change depending on how you are building this. For example, in a local environment I guess it should be CONDA_PREFIX, right?

Reply:
This is just used in the build stage (in `bld.bat`) so the environment variable should always be `PREFIX` as far as I understand.  I already put in way too many hours into this so I'm not that keen to investigate another solution with `gdalinfo` but I could do so if you think it really matters.

Reply:
Playing around with `gdalinfo`, it is for getting info on a file and I don't see any flags for giving info about the `libgdal` version itself.  Do you know of a secret flag for this?

Reply:
Sorry, `gdalinfo --version` will do the trick.  I just had a typo when I tried it the first time.  Okay, I'll look into that.

Reply:
@xylar no need. Like you said. You already put a lot of hours and this is good to go. Feel free to merge this and I'll open an issue to follow up later. No need to follow my crazy ideas ;-p

Reply:
@ocefpaf, sorry for being grouchy before. I did ask for input.  I spent a long time on this darn PR when I should have been doing other things, so that's what I'm grouchy about.

Your idea was a good one and not too hard to implement once I learned more than I wanted to know about tokenizing strings in `.bat` scripts.  Let's see how it goes...

Reply:
> @ocefpaf, sorry for being grouchy before. I did ask for input. I spent a long time on this darn PR when I should have been doing other things, so that's what I'm grouchy about.

There is no need to apologize when you are the one doing the work!

> Your idea was a good one and not too hard to implement once I learned more than I wanted to know about tokenizing strings in `.bat` scripts. Let's see how it goes...

That is mostly why I've been postponing it. Sorry to dump my ideas on your PRs. If this passes, let's merge, otherwise let's use your original idea. It is not worth to waste more of your time here.